nltk/draw/dispersion.py

```diff
@@ -15,51 +15,47 @@ def dispersion_plot(text, words, ignore_case=False, title="Lexical Dispersion Pl
     Generate a lexical dispersion plot.
 
     :param text: The source text
-    :type text: list(str) or enum(str)
+    :type text: list(str) or iter(str)
```

Comment:
`enum` in this context doesn't really make sense to me, but maybe that's my fault?